VirtoCommerce.JavaScriptShoppingCart.Web/Scripts/checkout/checkout-address.js

```diff
@@ -58,14 +58,15 @@ cartModule.component('vcCheckoutAddress', {
 				ctrl.address.regionName = undefined;
 			}
 		}
+		ctrl.setForm = function (frm) { ctrl.form = frm; };
 
 		ctrl.validate = function () {
 			if (ctrl.form) {
 				ctrl.form.$setSubmitted();
 				return !ctrl.form.$invalid;
 			}
 			return true;
-		}
+		};
 
 		function stringifyAddress(address) {
 			var stringifiedAddress = address.firstName + ' ' + address.lastName + ', ';
```

Comment:
I guess the code will be shorter in this context if use 'result' instead 'stringifiedAddress'. It's valid cuz this func doing only formating and nothing more... We know from the name of the function that the result is stringified and there are no reasons mark our result with this adjective

Reply:
fixed